staging.kernelci.org

```diff
@@ -245,10 +245,10 @@ cmd_docker() {
     ./kci docker $args kernelci
     ./kci docker $args k8s kernelci
 
-    ./kci docker $args api --version="$api_rev"
+    ./kci docker $args kernelci api --version="$api_rev"
     # TODO - add this functionality to kci docker
-    docker tag kernelci/staging-api:$api_rev kernelci/staging-api:latest
-    docker push kernelci/staging-api:latest
+    docker tag kernelci/staging-kernelci:api-$api_rev kernelci/staging-kernelci:api
```

Comment:
Should [API docker-compose](https://github.com/kernelci/kernelci-api/blob/50fca4be6a0c4755cb5dc239f1e454ff22c300fe/docker-compose.yaml#L11) be also adjusted to reflect this change?

Reply:
My bad, it already has been addressed as an update to https://github.com/kernelci/kernelci-api/pull/511